metagraph/integration_tests/test_query.py

```diff
@@ -32,10 +34,17 @@
 
 NUM_THREADS = 4
 
+def product(graph_types, anno_types):
+    result  = []
+    for graph in graph_types:
+        for anno in anno_types:
+            if graph == 'succinct' or (anno != 'row_diff' and anno != 'brwt_row_diff'):
+                result.append((graph, anno))
+    return result
 
 def build_annotation(graph_filename, input_fasta, anno_repr, output_filename, extra_params=''):
     target_anno = anno_repr
-    if anno_repr in {'rb_brwt', 'brwt'}:
+    if anno_repr in {'rb_brwt', 'brwt', 'row_diff', 'brwt_row_diff'}:
```

Comment:
I think it's better when the names of things from the same group share their prefix and not suffix.
For example, String1, String2, ... instead of FirstString, SecondString, etc.

The grouping here would be row_diff, which specifies what type of matrix is stored in the annotation.

Reply:
yeah, I would put the row-diff/no-row-diff on the higher place because that defined the content of the matrix. The representation is conceptually less important, it just defines compression.

Like `bit_vector_sd`, and not `sd_bit_vector`.
Don't ask me about `SuccinctDBG`... :)

Reply:
Ah, actually it's `DBGSuccinct`, so all food. The representation/compression is in the suffix.